---
title: "About"
---

I am a biologist and aspiring computational biologist who cultures stem cells and grow them into 3D models that mimic our body organs. Specifically, my previous project focused on the spatial distribution of gene expression levels within our forebrain. I utilized technologies such as single-cell RNA sequencing to map thousands of genes to their exact locations in tissue. The analysis involves the clustering of gene expression patterns to identify different cell types or regions within the tissue that share functional characteristics. Here is one of my favourite tissue sections where you can see clear neuron and progenitor zones.

![](About/Images/Screenshot%202023-12-05%20at%2011.24.22%E2%80%AFPM.png){fig-align="left"}

By Tobler's first law of geography, evrything is related to everything else, but near things are moew related than distant things. This emphsize the importance of spatial context. In spatial transcriptomics, the location of gene expression provides context that can be critical for understanding biological processes. In geospatial analysis, the location of physical features or events is key to understanding patterns and impacts on the environment and society.

I love to identify and interpret patterns within spatially organized data. Whether it's the clustering of gene expression profiles or the clustering of buses in urban areas, recognizing these patterns is central to extracting meaning from the data.

The fundamental importance of spatial relationships and the techniques used to analyze spatial data create a parallel between spatial transcriptomics and geospatial analysis. Understanding one can provide insights into the other, as both are ultimately about the significance of 'where'.
